packages/react-components/react-storybook-addon/.storybook/main.js

```diff
@@ -2,7 +2,7 @@ const rootMain = require('../../../../.storybook/main');
 
 module.exports = /** @type {Omit<import('../../../../.storybook/main'), 'typescript'|'babel'>} */ ({
   ...rootMain,
-  stories: [...rootMain.stories, '../stories/**/*.stories.mdx', '../stories/**/index.stories.@(ts|tsx)'],
+  stories: [...rootMain.stories, '../stories/**/*.stories.mdx', '../stories/**/*.stories.@(ts|tsx)'],
```

Comment:
the story for this addon doesn't need to be included in our docs, so switching this package to import non index stories, and changed the filename of the local story